en/pe-docs/docs/infrastructure/dataplanes/dataplanes.md

```diff
@@ -0,0 +1,22 @@
+# Data planes
+
+The data plane in Choreo is the runtime environment where user applications are deployed based on configurations defined in the control plane. It supports a variety of applications, including services, web apps, APIs, integrations, and scheduled tasks, and allows for a polyglot approach with multiple programming languages. All runtime traffic is securely contained within the data plane, ensuring user data remains isolated and protected.
+
+See Data planes concept for more information on Choreo's data plane architecture.
```

Comment:
Better if have the link?

Reply:
We should. I was waiting until the file structure is completed.
Added with [c0ad2be](https://github.com/wso2/docs-choreo-dev/pull/1736/commits/c0ad2be546a3fc38f4a28896f0674210d38878ed)